src/components/Home/Events/Events.css

```diff
@@ -69,16 +72,16 @@
 }
 
 .child {
```

Comment:
@31adityakumar child is a common class name try to change it to something specific like event_section_Child